src/TLuaInterpreter.h

```diff
@@ -399,6 +396,14 @@ class TLuaInterpreter : public QThread
     static int getProfileName( lua_State * );
     static int raiseGlobalEvent( lua_State * );
 
+public slots:
+    void slot_replyFinished( QNetworkReply * );
+    void slotPurge();
+    void slotDeleteSender();
+
+private:
+    TGatekeeperThread* mpGatekeeperThread;
+    QNetworkAccessManager* mpFileDownloader;
```

Comment:
Technically the Qt recommendations are for a single `QNetworkAccessManager` for the whole application (the Qt Documentation for it reads:
> The Network Access API is constructed around one `QNetworkAccessManager` object, which holds the common configuration and settings for the requests it sends. It contains the proxy and cache configuration, as well as the signals related to such issues, and reply signals that can be used to monitor the progress of a network operation. One `QNetworkAccessManager` should be enough for the whole Qt application.

We spawn up to three I think for each profile... :open_mouth: